src/components/ContentPost/ContentPost.tsx

```diff
@@ -1,58 +1,45 @@
 import { IPublication } from "@/utils/interfaces";
-import {
-  ContentWrapper,
-  Link,
-  Thumb,
-  Title,
-  TitleWrapper,
-  ArrowIcon,
-  DateWrapper,
-  Header,
-  Type,
-  ThumbContainer,
-  DescriptionContainer,
-  Description,
-} from "./ContentPost.styles";
 import { POST_TYPES } from "@/utils/constants";
+import Link from "next/link";
+import { Icon } from "@/components/Icon/Icon";
 
 const ContentPost = ({ content }: { content: { fields: IPublication } }) => {
-  const { title, thumb, link, date, type, description } = content.fields;
+  const { title, thumb, link, date, type } = content.fields;
   const dateObj = date ? new Date(date) : null;
   const formattedDate = dateObj ? dateObj.toLocaleDateString("pt-BR") : "";
 
   return (
-    <ContentWrapper>
-      <Header>
-        <Type>
-          {
-            POST_TYPES[
-              type as "additional-content" | "data-panel" | "newsletter"
-            ]
-          }
-        </Type>
-        <DateWrapper>{formattedDate}</DateWrapper>
-      </Header>
-      <Link href={link}>
-        <div>
-          <ThumbContainer>
-            <Thumb
-              src={`https:${thumb.fields.file.url}`}
-              alt=""
-              width={600}
-              height={300}
-            />
-            <DescriptionContainer>
-              <Description>{description || "Acessar postagem"}</Description>
-            </DescriptionContainer>
-          </ThumbContainer>
+    <Link
+      href={link}
+      className="flex flex-col overflow-hidden rounded-md w-full bg-grey-100 hover:bg-grey-200 border border-grey-200 hover:border-grey-300 cursor-pointer transition duration-300 shadow-sm"
+    >
+      <div className="relative group">
+        <img
+          alt=""
+          src={`https:${thumb.fields.file.url}`}
+          className="h-[180px] w-full block object-cover transition-transform group-hover:scale-102 duration-300"
+        />
+        <div className="absolute top-4 left-4 rounded-lg px-2 py-0.5 bg-gray-100">
+          <p className="font-semibold text-xs">
+            {
+              POST_TYPES[
+                type as "additional-content" | "data-panel" | "newsletter"
+              ]
+            }
+          </p>
+        </div>
+      </div>
 
-          <TitleWrapper>
-            <Title>{title}</Title>
-            <ArrowIcon id={"link-arrow"} size={16} />
-          </TitleWrapper>
+      <div className="p-5 flex justify-between items-center md:max-h-[76px]">
+        <div className="flex flex-col gap-0.5 md:max-w-[95%]">
+          <p className="line-clamp-2 leading-tight font-semibold text-base text-wrap">
+            {title}
+          </p>
+          <p className="text-[#7E797B] text-xs">{formattedDate}</p>
```

Comment:
Changed the text size to the one on figma and the color to the one in the palette on `global.css`

```suggestion
          <p className="text-grey-600 text-[10px]">{formattedDate}</p>
```